internal/infrastructure/host/proxy_infra.go

```diff
@@ -82,6 +82,18 @@ func (i *Infra) CreateOrUpdateProxyInfra(ctx context.Context, infra *ir.Infra) e
 	return nil
 }
 
+// getTopologyInjectorDisabled checks whether the provided EnvoyGateway disables TopologyInjector
+func getTopologyInjectorDisabled(gw *egv1a1.EnvoyGateway) bool {
```

Comment:
can we move this under as a internal method in https://github.com/envoyproxy/gateway/blob/main/api/v1alpha1/envoygateway_helpers.go

Reply:
Updated